### PR TITLE
[Outlook] (compose) Update samples

### PR DESCRIPTION
**Pull request by samantharamon**
- Updates samples to remove the requirement to configure the **\<Rule>** element in the manifest. 
- Reorganizes information and applies minor formatting fixes.

### CODE REVIEW

File: docs/outlook/get-set-or-add-recipients.md [hunk 1]
```diff
@@ -54,33 +47,26 @@ Note that because the `getAsync` method is asynchronous, if there are subsequent
...
+    if (info.host === Office.HostType.Outlook) {
+        item = Office.context.mailbox.item;
         getAllRecipients();
-    });
-}
+    }
+});
 
-// Get the email addresses of all the recipients of the composed item.
+// Get the email addresses of all the recipients of the item being composed.
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/get-set-or-add-recipients.md:64]:
```suggestion
// Gets the email addresses of all the recipients of the item being composed.
```

File: docs/outlook/insert-data-in-the-body.md [hunk 2]
```diff
@@ -1,206 +1,161 @@
...
+        item = Office.context.mailbox.item;
         setItemBody();
-    });
-}
+    }
+});
 
-// Get the body type of the composed item, and set data in 
-// in the appropriate data type in the item body.
+// Insert data at the current cursor position.
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/insert-data-in-the-body.md:48]:
```suggestion
// Inserts data at the current cursor position.
```

File: docs/outlook/insert-data-in-the-body.md [hunk 3]
```diff
@@ -1,206 +1,161 @@
...
-                         });
-                }
-            }
-        });
-}
 
-// Writes to a div with id='message' on the page.
-function write(message){
-    document.getElementById('message').innerText += message; 
+// Prepend data to the body of the item being composed.
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/insert-data-in-the-body.md:114]:
```suggestion
// Prepends data to the body of the item being composed.
```

**Review by ElizabethSamuel-MSFT — COMMENTED** [review]
How I was thinking about the comments for the functions. If a comment is describing what the function does, then I'll usually add an 's' e.g., "[This function] Does whatever.". Comments within a function I consider to be explaining is about to do e.g. "[run this code to] Do something." The stuff in the [] are unspoken/unwritten.

File: docs/outlook/get-or-set-the-subject.md [hunk 4]
```diff
@@ -1,122 +1,113 @@
...
 ```js
 let item;
 
-Office.initialize = function () {
-    item = Office.context.mailbox.item;
-    // Checks for the DOM to load using the jQuery ready method.
-    $(document).ready(function () {
-        // After the DOM is loaded, app-specific code can run.
-        // Set the subject of the item being composed.
+// Confirm that the Office.js library is loaded.
```
Comment (samantharamon) [inline docs/outlook/get-or-set-the-subject.md:78]:
```suggestion
// Confirms that the Office.js library is loaded.
```

File: docs/outlook/get-or-set-the-time-of-an-appointment.md [hunk 5]
```diff
@@ -1,129 +1,114 @@
...
 ```js
 let item;
 
-Office.initialize = function () {
-    item = Office.context.mailbox.item;
-    // Checks for the DOM to load using the jQuery ready method.
-    $(document).ready(function () {
-        // After the DOM is loaded, app-specific code can run.
-        // Get the start time of the item being composed.
+// Confirm that the Office.js library is loaded.
```
Comment (samantharamon) [inline docs/outlook/get-or-set-the-time-of-an-appointment.md:38]:
```suggestion
// Confirms that the Office.js library is loaded.
```

File: docs/outlook/get-or-set-the-time-of-an-appointment.md [hunk 6]
```diff
@@ -1,129 +1,114 @@
...
+        write(`The start time in UTC is: ${asyncResult.value.toString()}`);
+        // Convert the start time to local time and display it on the page.
+        write(`The start time in local time is: ${asyncResult.value.toLocaleString()}`);
+    });
 }
 
-// Write to a div with id='message' on the page.
-function write(message){
-    document.getElementById('message').innerText += message; 
+// Write to a div with id="message" on the page.
```
Comment (samantharamon) [inline docs/outlook/get-or-set-the-time-of-an-appointment.md:61]:
```suggestion
// Writes to a div with id="message" on the page.
```

File: docs/outlook/get-or-set-the-time-of-an-appointment.md [hunk 7]
```diff
@@ -1,129 +1,114 @@
...
 ```js
 let item;
 
-Office.initialize = function () {
-    item = Office.context.mailbox.item;
-    // Checks for the DOM to load using the jQuery ready method.
-    $(document).ready(function () {
-        // After the DOM is loaded, app-specific code can run.
-        // Set the start time of the item being composed.
+// Confirm that the Office.js library is loaded.
```
Comment (samantharamon) [inline docs/outlook/get-or-set-the-time-of-an-appointment.md:83]:
```suggestion
// Confirms that the Office.js library is loaded.
```

File: docs/outlook/get-set-or-add-recipients.md [hunk 8]
```diff
@@ -54,33 +47,26 @@ Note that because the `getAsync` method is asynchronous, if there are subsequent
...
 ```js
 let item;
 
-Office.initialize = function () {
-    item = Office.context.mailbox.item;
-    // Checks for the DOM to load using the jQuery ready method.
-    $(document).ready(function () {
-        // After the DOM is loaded, app-specific code can run.
-        // Get all the recipients of the composed item.
+// Confirm that the Office.js library is loaded.
```
Comment (samantharamon) [inline docs/outlook/get-set-or-add-recipients.md:56]:
```suggestion
// Confirms that the Office.js library is loaded.
```

File: docs/outlook/get-set-or-add-recipients.md [hunk 9]
```diff
@@ -89,101 +75,87 @@ function getAllRecipients() {
...
 ```js
 let item;
 
-Office.initialize = function () {
-    item = Office.context.mailbox.item;
-    // Checks for the DOM to load using the jQuery ready method.
-    $(document).ready(function () {
-        // After the DOM is loaded, app-specific code can run.
-        // Set recipients of the composed item.
+// Confirm that the Office.js library is loaded.
```
Comment (samantharamon) [inline docs/outlook/get-set-or-add-recipients.md:145]:
```suggestion
// Confirms that the Office.js library is loaded.
```

File: docs/outlook/get-set-or-add-recipients.md [hunk 10]
```diff
@@ -192,108 +164,107 @@ function setRecipients() {
...
+If you don't want to overwrite any existing recipients in an appointment or message, instead of using `Recipients.setAsync`, use the `Recipients.addAsync` asynchronous method to append recipients. `addAsync` works similarly as `setAsync` in that it requires a `recipients` input argument. You can optionally provide a callback function, and any arguments for the callback using the `asyncContext` parameter. Then, check the status, result, and any error of the asynchronous `addAsync` call using the `asyncResult` output parameter of the callback function. The following example checks if the item being composed is an appointment, then appends two required attendees to it.
 
 ```js
-// Add specified recipients as required attendees of
-// the composed appointment. 
+let item;
+
+// Confirm that the Office.js library is loaded.
```
Comment (samantharamon) [inline docs/outlook/get-set-or-add-recipients.md:239]:
```suggestion
// Confirms that the Office.js library is loaded.
```

File: docs/outlook/insert-data-in-the-body.md [hunk 11]
```diff
@@ -1,206 +1,161 @@
...
 let item;
 
-Office.initialize = function () {
-    item = Office.context.mailbox.item;
-    // Checks for the DOM to load using the jQuery ready method.
-    $(document).ready(function () {
-        // After the DOM is loaded, app-specific code can run.
-        // Insert data in the top of the body of the composed 
-        // item.
+// Confirm that the Office.js library is loaded.
```
Comment (samantharamon) [inline docs/outlook/insert-data-in-the-body.md:105]:
```suggestion
// Confirms that the Office.js library is loaded.
```

File: docs/outlook/insert-data-in-the-body.md [hunk 12]
```diff
@@ -1,206 +1,161 @@
...
 ```js
 let item;
 
-Office.initialize = function () {
-    item = Office.context.mailbox.item;
-    // Checks for the DOM to load using the jQuery ready method.
-    $(document).ready(function () {
-        // After the DOM is loaded, app-specific code can run.
-        // Set data in the body of the composed item.
+// Confirm that the Office.js library is loaded.
```
Comment (samantharamon) [inline docs/outlook/insert-data-in-the-body.md:40]:
```suggestion
// Confirms that the Office.js library is loaded.
```

File: docs/outlook/get-or-set-the-subject.md [hunk 13]
```diff
@@ -98,14 +98,14 @@ function setSubject() {
                 return;
             }
 
-            /*
+            The subject was successfully set.
               Run additional operations appropriate to your scenario and
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/get-or-set-the-subject.md:102]:
```suggestion
            /*
              The subject was successfully set.
              Run additional operations appropriate to your scenario and
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**